components/automate-ui/src/app/pages/project/rules/project-rules.component.html

```diff
@@ -77,26 +77,24 @@
               <chef-form-field class="attribute-field">
                 <label>
                   <span class="label">{{ getAttributeLabel() }}</span>
-                  <select ngDefaultControl formControlName="attribute" data-cy="attribute-dropdown">
-                    <option
-                      *ngFor="let option of attributes[ruleForm.get('type').value.toLowerCase()]; let i = index"
-                      [value]="option.key"
-                    >{{ option.value }}</option>
-                  </select>
+                  <chef-select ngDefaultControl formControlName="attribute" data-cy="attribute-dropdown" (change)="patchAttributeValue(ruleForm, $event, i)">
+                    <chef-option *ngIf="selectAttributeChefOption && currentRuleIndex !== i">Select...</chef-option>
+                    <chef-option *ngFor="let option of attributes[ruleForm.get('type').value.toLowerCase()]; let i = index"
+                    value="option.key">{{ option.value }}</chef-option>
```

Comment:
Would prefer to see complex logic kept out of the template, so suggest replacing L82-83 with:
```suggestion
<chef-option *ngFor="let option of attributes" value="option.key">{{ option.value }}</chef-option>
```
where you define the `attributes` property in the backing code.
That has the added benefit of this line then mirroring L96 below.
(And I do not believe the `let i = index` portion there is used at all.)

Reply:
@msorens I have updated the changes please have a look.